internal/controller/nginx/config/stream_servers_test.go

```diff
@@ -411,3 +411,86 @@ func TestCreateStreamServersWithNone(t *testing.T) {
 
 	g.Expect(streamServers).To(BeNil())
 }
+
+func TestExecuteStreamServersWithResolver(t *testing.T) {
```

Comment:
I noticed that in `base_http_config_test.go` the function names use Underscores like with `TestExecuteBaseHttp_DNSResolver`

Should we be looking to keep those consistent? Not looking to change that here, but just want to open the discussion.